README.md

```diff
@@ -2,9 +2,7 @@
 [![Paper shield](https://img.shields.io/badge/JMLR-Paper-red)](http://www.jmlr.org/papers/volume20/19-490/19-490.pdf)
 [![PyPI version](https://img.shields.io/pypi/v/graspologic.svg)](https://pypi.org/project/graspologic/)
 [![Downloads shield](https://pepy.tech/badge/graspologic)](https://pepy.tech/project/graspologic)
-[![Docs shield](https://img.shields.io/readthedocs/graspologic)](https://graspologic.readthedocs.io/)
 ![graspologic CI](https://github.com/microsoft/graspologic/workflows/graspologic%20CI/badge.svg)
-[![codecov](https://codecov.io/gh/microsoft/graspologic/branch/dev/graph/badge.svg)](https://codecov.io/gh/microsoft/graspologic)
```

Comment:
I think this is not turned on currently?

Reply:
https://github.com/microsoft/graspologic/blob/dev/.github/workflows/build.yml#L86 I think it is?

Reply:
I don't know how it actually works, but I know we're running code coverage as part of the build

Reply:
And by "I know" I mean "I ... guess?"

Reply:
https://app.codecov.io/gh/microsoft/graspologic this looks not right to me

Reply:
possibly different from the check you mention though

Reply:
yeah i think the actual pytest cov thing is running, but this app that we used to use for reporting the results is not. at least that is my understanding

Reply:
We do run code coverage but it does not seem to send the results to codecov.io.  The code coverage is so low that we look better by not reporting it :).

Reply:
Why oh why does this UI not refresh so I don't repeat what everyone else said? :)